ts/visitor.ts

```diff
@@ -544,15 +544,25 @@ export const visitor = {
       }
     }
   },
+  SwitchStatement(path: NodePath<t.SwitchStatement>, st: S) {
```

Comment:
This needs more thought. As written, this will prevent breaks in braces. Is that something we want? See #46 for follow-up problem spec discussion.

Reply:
So what restrictions are there on the contents of case blocks (if any) aside from they must end in `break;` or have one immediately after?